t_cam: [0, 0, 0.02]
cam_to_quad_rot: [0,0,1,-1,0,0,0,-1,0]
camera_matrix: [415.21159208661874, 0.0, 424.5, 0.0, 415.21159208661874, 240.5, 0.0, 0.0, 1.0]
verbose: true
image_height: 480
image_width: 848
